### ISSUE TITLE
if there is keyfile , plz support root@10.0.0.1 with out -credential

### ISSUE THREAD

**Issue by guijunhua** [darkoperator/Posh-SSH#325]
i want use it write ps script 
is there any way ,if i have  keyfile ,Just  not use -credential, uesr name is enough .

**Comment by barky11** [comment]
If you want to use a username and keyfile, you still need to use -credential , but you leave your password variable empty. Here are the commands.
$username="root"
$nopasswd = new-object System.Security.SecureString
$credential = New-Object System.Management.Automation.PSCredential ($username,$nopasswd)
New-SSHSession -computername <IP/Hostname> -credential $credential -Keyfile <Keyfile>

**Comment by darkoperator** [comment]
That is a very very bad practice to have keys without a passphrase to protect them

**Comment by dreamorosi** [comment]
@darkoperator 100% agree with you, but I think it should be up to the user to enforce this. 

We don't always create/control the key unfortunately and it would be useful if this use case would be supported, perhaps with a notice in the documentation that says "this is not advised".

**Comment by darkoperator** [comment]
Yep should document it better, the bad practice of no passphrase is used a lot for keys, even for root. Seen it abused a lot in recent ransomware attacks and some crypto miner malware campaigns, did not thought it was so widespread.
